sdk/translation/Azure.AI.Translation.Document/CHANGELOG.md

```diff
@@ -1,14 +1,16 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2024-05-07)
```

Comment:
You also have to update this in the csproj file so that the package gets released with this version.

Reply:
@joseharriaga is csproj the only place we need to update the package version?